plugins/nf-schema/src/test/nextflow/validation/NfValidationTest.groovy

```diff
@@ -37,6 +42,11 @@ class NfValidationTest extends Dsl2Spec{
             protected PluginDescriptorFinder createPluginDescriptorFinder() {
                 return new TestPluginDescriptorFinder(){
                     @Override
+                    protected Manifest readManifestFromDirectory(Path pluginPath) {
```

Comment:
are these functions used by Nextflow?

Reply:
Nope they are from the groovy testing library, I used this test as inspiration: https://github.com/edn-es/nf-pgcache/blob/main/src/test/groovy/es/edn/nextflow/pgcache/PluginTest.groovy

This was needed because the previous implementation was causing issue with the new version of `groovy-test`